packages/web-runtime/tests/unit/pages/resolvePublicLink.spec.ts

```diff
@@ -34,29 +34,40 @@ describe('resolvePublicLink', () => {
   })
   describe('password required form', () => {
     it('should display if password is required', async () => {
-      const { wrapper } = getWrapper({ passwordRequired: true })
+      const { wrapper } = getWrapper({ passwordRequired: true }) as any
```

Comment:
In general, I am not a fan of `any` keyword. Why is it needed and would there be some easy way around it?